primerj/src/main/java/org/primer/primerj/utils/BlockUtil.java

```diff
@@ -169,21 +169,27 @@ public synchronized static Block dowloadSpvBlock() throws Exception {
                 blockChainGetLatestBlockApi.handleHttpGet();
                 block = blockChainGetLatestBlockApi.getResult();
                 log.info("Block: " + block + " ");
-                log.info("Block: interval is " + String.valueOf(PrimerjSettings.INTERVAL));
             }
         } catch (Exception e) {
             e.printStackTrace();
             AbstractApp.notificationService.sendBroadcastGetSpvBlockComplete(false);
             throw e;
         }
+
+        /*
```

Comment:
Generally should always delete code instead of keeping them in comments.

Reply:
As suggested, the old code has been deleted.